services/docker-stack-bo-ps-corrs.yml

```diff
@@ -115,50 +75,30 @@ services:
       - ioc-network
 
   ia07:
-      image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:__FAC_IOC_TAG_TEMPLATE__
-      command: bash -c 'source /etc/profile && /ioc-apps/bo-ps-corrs-ia07.bash'
-      deploy:
-        placement:
-          constraints:
-            # - node.hostname != CA-RaTim-CO-IOCSrv
-            # - node.hostname != IA-14RaDiag03-CO-IOCSrv
-            # - node.hostname != IA-16RaBbB-CO-IOCSrv
-            # - node.hostname != IA-18RaDiag04-CO-IOCSrv
-            # - node.hostname != IA-20RaDiag01-CO-IOCSrv-1
-            - node.hostname != IA-20RaDiag01-CO-IOCSrv-2
-            # - node.hostname != IA-20RaDiag02-CO-IOCSrv-1
-            # - node.hostname != IA-20RaDiag02-CO-IOCSrv-2
-            - node.hostname != CA-RaCtrl-CO-Srv-1
-            - node.hostname != LA-RaCtrl-CO-Srv-1
-            - node.hostname != TA-TiRack-CO-FWSrv-1
-        replicas: 1
-        restart_policy:
-          condition: any
-      logging:
-        driver: "json-file"
-        options:
-          max-file: "10"
-          max-size: "10m"
-      networks:
-        - ioc-network
+    image: dockerregistry.lnls-sirius.com.br/fac/fac-iocs:__FAC_IOC_TAG_TEMPLATE__
+    command: bash -c 'source /etc/profile && /ioc-apps/bo-ps-corrs-ia07.bash'
+    deploy:
+      placement:
+        constraints:
+          - node.hostname == lnlsfac-srv2
+      replicas: 1
+      restart_policy:
+        condition: any
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "10m"
+    networks:
+      - ioc-network
```

Comment:
is this alignment correct?

Reply:
new versions of these files were generated with a script (one of the new files in this pr). they all should have the exact same format.

Reply:
Oh, I see. So probably the indentation change here is a fix.